profile/README.md

```diff
@@ -0,0 +1,26 @@
+# Welcome to the BonvinLab GitHub page
+
+![bonvinlab-banner](banner.png)
+
+The BonvinLab is a Computational Structural Biology group that focuses on dissecting, understanding and predicting biomolecular interactions at the molecular level. Our research is focused on the development of integrative computational methods for the study of biomolecular interactions, with a particular emphasis on the structural characterization of protein-protein and protein-ligand complexes.
+
+We are part of the [Bijvoet Center for Biomolecular Research](https://www.uu.nl/en/research/bijvoet-centre-for-biomolecular-research) at [Utrecht University](https://www.uu.nl/),  affiliated with the [Netherlands eScience Center](https://www.esciencecenter.nl/) and one of the core applications of the [BioExcel Center of Excellence for Computational Biomolecular Research](https://bioexcel.eu/) under [EuroHPC](https://eurohpc-ju.europa.eu/index_en).
+
+## Featured research software
+
+- [haddock v3](https://github.com/haddocking/haddock3): The  **H**igh **A**mbiguity **D**riven biomolecular **DOCK**ing is our flagship software, it is an integrative platform for modelling biomolecular complexes. It is one of the most popular software for protein-protein and protein-ligand docking.
+- [pdb-tools](https://github.com/haddocking/pdb-tools): A dependency-free cross-platform swiss army knife for PDB files.
+- [prodigy](https://github.com/haddocking/prodigy): Predict the binding affinity of protein-protein complexes from structural data
+- [arctic3d](https://github.com/haddocking/arctic3d): Automatic Retrieval and ClusTering of Interfaces in Complexes from 3D structural information
+- [proabc-2](https://github.com/haddocking/proABC-2): Deep learning framework to predict antibody paratope residues
```

Comment:
Since we do no longer provide the services, do we want it to be promoted here?  Could also add disvis

Reply:
I think it's good to promote it here, this banner is about the source-code - which is fine and working

Reply:
Ok!  What about DISVIS?